Tests/TensorFlowTests/OperatorTests/LinearAlgebraTests.swift

```diff
@@ -115,6 +115,22 @@ final class LinearAlgebraTests: XCTestCase {
         assertEqual(computedGradient, expectedGradient, accuracy: 1e-16)
     }
 
+    func testDet() {
+        let matrix = Tensor<Float>(shape: [1, 4, 4], scalars: (0..<16).map(Float.init))
+        let computedDet = det(matrix)
+        let expectedDet = Tensor<Float>([0])
```

Comment:
Could you please at one more test for `det` and `slogdet`, with different expected values?
I'd recommend using input tensors with shape `[2, 2, 4, 4]`.